modules/master-node-sizing.adoc

```diff
@@ -33,6 +33,63 @@ On a cluster with three masters or control plane nodes, the CPU and memory usage
 The node sizing varies depending on the number of nodes and object counts in the cluster. It also depends on whether the objects are actively being created on the cluster. During object creation, the control plane is more active in terms of resource usage compared to when the objects are in the `running` phase.
 ====
 
+Operator Lifecycle Manager (OLM ) runs on the master nodes and it's memory footprint depends on the number of namespaces and user installed operators that OLM needs to manage on the cluster. Master nodes need to be sized accordingly to avoid OOM kills. Following data points are based on the results from cluster maximums testing.
```

Comment:
s/it's/its

user-installed Operators that OLM must manage on the cluster.

Master nodes must be sized accordingly to avoid out-of-memory (OOM) failures. The following data points are based on the results of

Reply:
Fixed it. Thanks.